actions/release-it/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Release-it"
+description: "Execute release-it"
+author: "niek.vanstaveren@qlik.com"
+
+inputs:
+  GH_TOKEN:
+    description: ""
+    required: true
+  NPM_TOKEN:
+    description: ""
+    required: true
+  package_manager:
+    description: ""
+    required: true
+  monorepo_manager:
+    description: ""
+    required: false
+  release_branches:
+    description: ""
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Configure release-it and git user
+      id: git-setup
+      shell: bash
+      run: |
+        git config user.name "${GITHUB_ACTOR}"
+        git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+        git config --global url.https://${{ inputs.GH_TOKEN }}@github.com/.insteadOf https://github.com/
+        branch_name=$(git rev-parse --abbrev-ref HEAD)
+        echo "branch_name=$branch_name" >> "$GITHUB_OUTPUT"
+    - name: Release it
+      env:
+        GITHUB_TOKEN: ${{ inputs.GH_TOKEN }}
+        NPM_TOKEN: ${{ inputs.NPM_TOKEN }}
+        release_branches: ${{ inputs.release_branches }}
+      shell: bash
+      if: ${{ !inputs.monorepo_manager }}
+      run: ${{ inputs.package_manager }} release-it --config ${GITHUB_ACTION_PATH}/.release-it.js
+    - name: Release it monorepo
+      env:
+        GITHUB_TOKEN: ${{ inputs.GH_TOKEN }}
+        NPM_TOKEN: ${{ inputs.NPM_TOKEN }}
+        release_branches: ${{ inputs.release_branches }}
+        monorepo: ${{ !!inputs.monorepo_manager }}
+        branch_name: ${{ steps.git-setup.outputs.branch_name}}
+      shell: bash
+      if: ${{ inputs.monorepo_manager }}
+      run: ${{ inputs.package_manager }} workspaces run release-it --config ${GITHUB_ACTION_PATH}/.release-it.js -VV
```

Comment:
Should be possible to have only one release step here. I'll start testing `normal` repos now as well.